### ISSUE TITLE
EPUB WCAG compliance "Every HTML document needs a lang attribute"

### ISSUE THREAD

**Issue by juliusross1** [brucemiller/LaTeXML#2661]
Created an epub file using latexmlc and ran it through Ace by Daisy who indicate this violation of WCAG:

```
Serious: WCAG 2.0 A
html-has-lang
[Learn about: Language](https://github.com/brucemiller/LaTeXML/issues/new#)
The <html> element does not have a lang attribute 
 Ensure every HTML document has a lang attribute 
 <html> element must have a lang attribute
```

<img width="912" height="230" alt="Image" src="https://github.com/user-attachments/assets/1eebf00f-8b6e-4505-9e4d-04419edf84ce" />

I looked at the epub file and the lang is there in the content.opf file but is not included in the xhtml files.  

Somebody with more expertise than I have would need to say if this is actually required by WCAG, or if the Ace by Daisy is reporting a violation that is not really there.

**Comment by dginev** [comment]
Thank you for the report!

This is probably an oversight in our ePub output, as the HTML output already has a lang attribute set by default. We'll take a look.
